# controller gains (position)
ImpedanceControllerGainsMsg command
---
ImpedanceControllerGainsMsg command
bool accepted
